src/lib/features/listening-history/model/store.ts

```diff
@@ -1,8 +1,22 @@
 import { currentlyPlayingEpisode } from '$lib/entities/episode';
+import { supabaseClient, type Episode } from '$lib/shared/api';
+import { user } from '$lib/entities/user';
 import { localStorageAdapter, persistentWritable } from 'svelte-persistent-writable';
+import { get } from 'svelte/store';
+
+user.subscribe(async ($user) => {
+  if (!$user) return;
+  const hist = await supabaseClient
```

Comment:
suggestion (non-blocking): I think this code should go in the `api` segment. Also the code below that adds to the `history` table

Reply:
You resolved this conversation without a response, what does that mean?

Reply:
Probably confused it with another comment. Thank for pointing out.

Reply:
Done. Please, re-review this point. I hope to add corresponding tests soon.